webapp/src/shared/components/Text/Text.tsx

```diff
@@ -28,21 +29,29 @@ export enum TextWeight {
 type TextProps = {
   variant: TextVariant;
   color?: TextColor;
+  parent_class?: string;
   weight?: TextWeight;
   children: string;
 };
 
 export default function Text({
   variant,
-  color = TextColor.DARK,
+  color,
+  parent_class,
```

Comment:
I think you should merge first #96, and then we should revisit this, because we WILL have conflicts, and some of your changes are a implementing, in a different way, some of my changes.